cordova-plugin-outline/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "cordova-plugin-outline",
+  "version": "1.0.0",
```

Comment:
I wonder what the right value for "version" is in a case like this.  If this number doesn't actually increment when the code changes, "0.0.0" or "0" might be clearer.

Reply:
I was wondering too. We declared version 1.0.0 in plugin.xml since the start. I don't think it matters in practice, but happy to change it if you feel strongly about it.

Reply:
Just a suggestion.  If you agree, it's a one-character change, and easy to check that it still works.

If you prefer it the other way, that's fine.

Reply:
I like the suggestion :) Done in e8310c4.